docs/using.rst

```diff
@@ -381,11 +378,15 @@ be on a different computer.
 Nginx
 -----
 
-In the future, if you're running Nginx you will hopefully be able to use this
-plugin to automatically obtain and install your certificate. The Nginx plugin is
-still experimental, however, and is not installed with certbot-auto_. If
-installed, you can select this plugin on the command line by including
-``--nginx``.
+The Nginx plugin is included in Certbot 0.9.0. It should
```

Comment:
nit: To me this is a little misleading. The Nginx plugin is not included in version 0.9.0 of the Certbot package. I'm fine leaving this alone if you don't think it's a problem.

If you decide to make the change, you used the same wording in the table above.